quupe-RN/js/screens/Home/styles.js

```diff
@@ -0,0 +1,9 @@
+import { Dimensions, Platform, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  // dimension: {
+  //   margin: '3.5%'
+  // }
+});
```

Comment:
please remove the commented styling and do your styling here instead of the container file or presentation file.